other/materials_designer/create_perturbation.ipynb

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
<p>We should demonstrate how to build an arbitrary perturbation here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Now there's SineWave and arbitrary function examples, do you mean to leave only arbitrary one?